### PR TITLE
Update dynamic-prices.mdx

### PR DESCRIPTION
**Pull request by Gertjezzz**
Translated German to English

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/features/dynamic-prices.mdx [hunk 1]
```diff
@@ -61,38 +61,38 @@ tariffs:
...
 
 <Screenshot
   name="features/screenshots/dynamicprice-modal"
   caption="Screenshot vom Smartes Netzladen Dialog mit eingestelltem Preislimit"
 />
 
-Dieses Limit gilt im **PV-Modus** des aktuellen Ladepunktes.
-Mit Klick auf "überall anwenden" wird das Limit für alle Ladepunkte übernommen.
-Zusätzlich zum [PV-Überschussladen](./solar-charging) wird nun in den Stunden mit niedrigem Strompreis (grüne Balken) schnelles Laden aktiviert.
+This limit applies in **PV mode** of the current charging point.
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/features/dynamic-prices.mdx:83]:
```suggestion
This limit applies in **solar mode** of the current charging point.
```

File: i18n/en/docusaurus-plugin-content-docs/current/features/dynamic-prices.mdx [hunk 2]
```diff
@@ -10,17 +10,17 @@ import Screenshot from "../../../../../src/components/Screenshot";
 This page needs to be translated from German to English.
 :::
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/features/dynamic-prices.mdx:11]: The above warning can be removed.

File: i18n/en/docusaurus-plugin-content-docs/current/features/dynamic-prices.mdx [hunk 3]
```diff
@@ -77,10 +77,10 @@ Here you can see the energy prices for the coming hours and set a price limit.
 
 <Screenshot
   name="features/screenshots/dynamicprice-modal"
-  caption="Screenshot vom Smartes Netzladen Dialog mit eingestelltem Preislimit"
+  caption="Screenshot of the Smart Network Charging dialog with set price limit"
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/features/dynamic-prices.mdx:80]: "smart grid charging"

**Comment by naltatis** [comment]
@Gertjezzz thanks for your translation 💚

**Comment by Gertjezzz** [comment]
The title would be better called: "Dynamic Prices"

**Pull request merged**